2-repositories/scala-repo-server/project/BuildSettings.scala

```diff
@@ -22,7 +22,7 @@ object BuildSettings {
     organization            := "com.snowplowanalytics",
     version                 := "0.2.0",
     description             := "Scala schema server for Iglu",
-    scalaVersion            := "2.10.1",
+    scalaVersion            := "2.10.4",
```

Comment:
Current Scala version, 2.10.1, isn't compatible with JDK8 compiled classes, e.g. https://travis-ci.org/snowplow/iglu/builds/300073622#L1004 , thus I inserted a temporary bump in Scala version. Kafka had a similar issue [here](https://issues.apache.org/jira/browse/KAFKA-1624)

Reply:
That's right. But please, bump it to latest `2.10.6`.